src/components/Header/components/WalletIcon/icons/index.ts

```diff
@@ -0,0 +1,93 @@
+// Icons
+import metamaskIcon from './icon-metamask.png'
+import metamaskIcon2x from './icon-metamask@2x.png'
+import walletConnectIcon from './icon-wallet-connect.svg'
+import trezorIcon from './icon-trezor.svg'
+import ledgerIcon from './icon-ledger.svg'
+import dapperIcon from './icon-dapper.png'
+import dapperIcon2x from './icon-dapper@2x.png'
+import fortmaticIcon from './icon-fortmatic.svg'
+import portisIcon from './icon-portis.svg'
+import authereumIcon from './icon-authereum.png'
+import torusIcon from './icon-torus.svg'
+import uniloginIcon from './icon-unilogin.svg'
+import coinbaseIcon from './icon-coinbase.svg'
+import operaIcon from './icon-opera.png'
+import operaIcon2x from './icon-opera@2x.png'
+
+import { WALLET_PROVIDER } from 'src/logic/wallets/getWeb3'
+
+interface WalletObjectsProps<TValue> {
+  [key: string]: TValue
```

Comment:
The amount of wallets we have is defined and finite, so can we possibly make a type `type WalletProviderNames = keyof typeof WALLET_PROVIDER` or something like this?